recipe/build.sh

```diff
@@ -8,10 +8,14 @@ cmake ${CMAKE_ARGS} -GNinja \
   -DCMAKE_PREFIX_PATH=$PREFIX \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
   -DCMAKE_INSTALL_LIBDIR=lib \
-  -DENABLE_TESTING=OFF \
+  -DENABLE_TESTING=ON \
   -DCMAKE_BUILD_TYPE=Release \
   -DBUILD_SHARED_LIBS=ON \
+  -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
   -DUSE_OPENSSL=ON \
```

Comment:
If it is only for Linux libopenssl, maybe we can wrap it?
```suggestion
if [[ ${target_platform} == linux-* ]]; then
  -DUSE_OPENSSL=ON \
fi
```